server/pkg/controller/storagebonus/referral.go

```diff
@@ -141,6 +142,12 @@ func (c *Controller) UpdateReferralCode(ctx *gin.Context, userID int64, code str
 	if len(code) < 4 || len(code) > 20 {
 		return stacktrace.Propagate(ente.NewBadRequestWithMessage("code length should be between 4 and 8"), "")
 	}
+
+// Check if the code contains any offensive language using the go-away library
```

Comment:
@brog-io Indentation seems off, could you format these lines please? Thanks!

Reply:
Done